policy/exposures.yaml

```diff
@@ -5,6 +5,13 @@ deepCoadd:
     python: lsst.afw.image.ExposureF
     template: deepCoadd/%(filter)s/%(tract)d/%(patch)s.fits
     level: Skytile
+fakes_deepCoadd:
+    description: "A coadd assembled from direct warps, produced by AssembleCoaddTask, with fakes added."
```

Comment:
fakes => fake objects, breaks degeneracy between fake objects inserted in the warps and having fake warps included in the coadd